docs-ref-mapping/reference-unified.yml

```diff
@@ -267,26 +251,24 @@
     uid: azure.java.sdk.landingpage.services.containerregistry
     landingPageType: Service
     items:
-    - name: Management
-      uid: azure.java.sdk.landingpage.services.containerregistry.Management
-      landingPageType: Service
-      children:
-      - com.microsoft.azure.management.containerregistry*
     - name: Resource Management
       uid: azure.java.sdk.landingpage.services.containerregistry.ResourceManagement
       landingPageType: Service
       children:
-      - com.azure.resourcemanager.containerregistry*   
+      - com.azure.resourcemanager.containerregistry*  
+      - com.microsoft.azure.management.containerregistry* 
```

Comment:
is this still necessary?

Reply:
this service is not GA-ed yet so we put both Track 1 and Track 2 here still